scripts/install.js

```diff
@@ -92,37 +90,22 @@ async function cleanDeps() {
   await mkdir(depsPath);
 }
 
-/**
- * Symlinks the extracted libtensorflow library to the destination path. If the
- * symlink fails, a copy is made.
- */
-async function symlinkDepsLib() {
-  if (destLibPath === undefined) {
-    throw new Error('Destination path not supplied!');
-  }
-  try {
-    await symlink(depsLibPath, destLibPath);
-  } catch (e) {
-    console.error(
-        `  * Symlink of ${destLibPath} failed, creating a copy on disk.`);
-    await copy(depsLibPath, destLibPath);
-  }
-}
-
-/**
- * Moves the deps library path to the destination path.
- */
-async function moveDepsLib() {
-  if (destLibPath === undefined) {
-    throw new Error('Destination path not supplied!');
+async function verifyCUDA() {
+  const { stdout, stderr } = await exec('nvcc --version');
```

Comment:
This is nice - but I worry about it working on windows. Something to think about once we have Windows CUDA builds.

Reply:
I found similar command line tool for Windows CUDA version, though I haven't tried it yet. But I'm wondering how do you feel comfortable about using command line tool to get if the machine has CUDA GPU?